packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.controller.js

```diff
@@ -111,6 +111,20 @@ export default class {
       : this.fetchLegacyConsumption(datastores);
   }
 
+  fetchLocations(datastores) {
+    return this.$q.all(
+      datastores.map((datastore) => {
+        return this.ovhManagerPccDatacenterDatastoreService
+          .getDatastoreLocation(this.productId, datastore.dc, datastore.id)
+          .then((response) => {
+            set(datastore, 'location', response);
```

Comment:
we prefer not to use lodash any more

Reply:
Didn't know that, was keeping the same coding style as other files, removing it.